bsdploy/fabfile_digitalocean.py

```diff
@@ -33,20 +36,21 @@ def bootstrap(**kwargs):
     # overwrite sshd_config, because the DO version only contains defaults
     # and a line explicitly forbidding root to log in
     sudo("""echo 'PermitRootLogin without-password' > /etc/ssh/sshd_config""")
```

Comment:
oh, i didn't notice that. in that case, i'll incorporate your change and we just merge my pr?